include/pressio/ops/tpetra/ops_level2.hpp

```diff
@@ -67,6 +67,7 @@ namespace impl{
 template <class A_type, class x_type, class y_type, class alpha_t, class beta_t>
 ::pressio::mpl::enable_if_t<
   ::pressio::is_multi_vector_tpetra<A_type>::value and
+  ::pressio::is_vector_eigen<x_type>::value and
   ::pressio::is_vector_tpetra<y_type>::value
```

Comment:
this function is just an impl, and the name already conveys what it is doing, I think we can remove these two lines
```cpp
//remove this ::pressio::is_multi_vector_tpetra<A_type>::value and 
//remove this ::pressio::is_vector_eigen<x_type>::value and
::pressio::is_vector_tpetra<y_type>::value
```

Reply:
we can leave this for now , not a blocker